modules/docs/docgen/spec/docParser.spec.ts

```diff
@@ -0,0 +1,1419 @@
+import {createProgramFromSource} from './createProgramFromSource';
+import {parse} from '../docParser';
+
+describe('docParser', () => {
+  describe('simple values', () => {
+    it('should find an exported type of string', () => {
```

Comment:
This describes the following spec (same description, not accurate here). It should probably be:
```suggestion
    it('should find an exported type of primitive', () => {
```

Reply:
I also notice some of these descriptions will read like this: "should find an exported type of a primitive" but the "a" is not consistently used, i.e. it says "should find an exported type of primitive." There's a different semantic meaning with or without the "a". Is this correct for each of these descriptions?

Reply:
I'll fix